src/displayapp/screens/WatchFacePineTimeStyle.cpp

```diff
@@ -42,6 +42,41 @@ namespace {
     auto* screen = static_cast<WatchFacePineTimeStyle*>(obj->user_data);
     screen->UpdateSelected(obj, event);
   }
+
+  const char* GetIcon(const Pinetime::Controllers::SimpleWeatherService::Icons icon) {
```

Comment:
This is quite a universal function, so I think it'd be best to put this somewhere that other apps that want to display weather can access.

Reply:
Yes, I moved it to a new file!